internal/searcher/searcher.go

```diff
@@ -9,11 +9,12 @@ import (
 	"strings"
 
 	"github.com/samber/lo"
+	"go.jetpack.io/devbox/internal/boxcli/usererr"
 )
 
 func SearchAndPrint(w io.Writer, query string) error {
 	c := NewClient()
-	result, err := c.Search(query)
+	result, err := c.Search(query, "" /*version*/)
```

Comment:
What about having `Search(query string)` and `SearchVersion(query, version string)` to make the version parameter more obvious?